### PR TITLE
Reroutes all routes to path routes

### PR DESCRIPTION
**Pull request by ZooeyMiller**
All routes in the `/admin` and `/login` scopes now redirect to pages in the `page` scope, so that we don't have any unnecessary repetition. 

#60

### CODE REVIEW

File: lib/library_web/controllers/admin_controller.ex [hunk 1]
```diff
@@ -38,6 +24,6 @@ defmodule LibraryWeb.AdminController do
     Books.get_book!(id)
     |> Books.delete_book_and_unique_authors
 
-    render(conn, "index.html", books: [])
+    redirect(conn, to: page_path(conn, :index))
```
Comment (katbow) [inline lib/library_web/controllers/admin_controller.ex]: You originally had `books: []` passed in as well, do you still need this?

Comment (finnhodgkin) [reply lib/library_web/controllers/admin_controller.ex]: I think index path defaults to displaying all the books

File: lib/library_web/router.ex [hunk 2]
```diff
@@ -28,16 +28,13 @@ defmodule LibraryWeb.Router do
   scope "/admin", LibraryWeb do
     pipe_through :browser
 
-    get "/", AdminController, :index
-    get "/search", AdminController, :search
     post "/create", AdminController, :create
     get "/delete/:id", AdminController, :delete
   end
```
Comment (katbow) [inline lib/library_web/router.ex]: I didn't catch this earlier. Can you change this to use `resources "/" AdminController, only [:create, :delete]`? 
Then we'll end up with `admin_path  POST    /admin           Library.AdminController :create` and `admin_path  DELETE    /admin           Library.AdminController :delete`.

[More on resources](https://hexdocs.pm/phoenix/Phoenix.Router.html#resources/4).

Comment (ZooeyMiller) [reply lib/library_web/router.ex]: This makes a lot of sense, could we raise it as a separate issue and work on it in a different PR?

Comment (katbow) [reply lib/library_web/router.ex]: Yes I think that's best actually, since there are a few other things we can go over for the router 👍

**Pull request merged**